apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V94__new_schema_final_cleanup.sql

```diff
@@ -0,0 +1,57 @@
+-- NEW_SCHEMA_FINAL-CLEANUP.SQL
+--
+-- This flyway script is intended to a perform final cleanup of BFD database
+-- schema changes related to using BIGINT for bene_id, clm_id, and clm_grp_id.
+--
+-- In particular, this script:
+--   1) fixes sppelling errors in constraint names
+--   2) removes some unused table indeces.
+--   3) drop db triggers that are no longer used and in fact are invalid.
+--   4) drops some tables that were temporarily used during various data mitagation efforts
+--   5) drop unused procedures that were used from db trigger(s)
+--
+-- fix spelling error 'benficiaries' in constraint names
+--
+${logic.psql-only} alter table if exists public.medicare_beneficiaryid_history
+${logic.psql-only}  rename constraint medicare_beneficiaryid_history_to_benficiaries
+${logic.psql-only}    to medicare_beneficiaryid_history_to_beneficiaries;
+
+${logic.psql-only} alter table if exists public.beneficiary_monthly
+${logic.psql-only}  rename constraint beneficiary_monthly_to_benficiaries
+${logic.psql-only}    to beneficiary_monthly_to_beneficiaries;
+
+-- remove unused indexes from beneficiaries table
+drop index if exists  beneficiaries_ptd_cntrct_number_jan_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_feb_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_mar_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_apr_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_may_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_jun_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_jul_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_aug_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_sept_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_oct_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_nov_id_idx;
+drop index if exists  beneficiaries_ptd_cntrct_number_dec_id_idx;
+
+-- drop unused / invalid db triggers.
+${logic.psql-only} drop trigger ${logic.psql-only} if exists
```

Comment:
does the first psql-only not affect the whole line? Why is the second one in there?

Reply:
yea, altho it doesn't affect anything other than to prove I am an idiot! the triggers were created (by me) as part of the CCW remediation in beginning of 2021. At first I had the script doing this for hsql, but realized that I didn't need that for hsql since it was never created....then forgot to remove 2nd macro! Thx.